### ISSUE TITLE
[base-cluster] user management

### ISSUE THREAD

**Issue by cwrau** [teutonet/teutonet-helm-charts#99]
We should be able to manage various user accounts via the helm chart

we need the following;

- [x] list of user accounts
- [x] (Cluster-)Roles?
- [x] (Cluster-)Rolebindings?
- [ ] Namespaces?

~We probably can't automate the creation of the tokens because of the "recent" changes in k8s v1.23~

We can just create the secret manually 🤷

~Therefore~ Still, we should print a command that generates the kubeconfig, including the creation of the token

**Comment by cwrau** [comment]
@marvinWolff  do we need the unchecked items?

**Comment by marvinWolff** [comment]
Hm, good question.

What Roles and Rolebindings were used in the old cluster?
But anyway, i think it's hard to make general assumptions about what roles are needed for a specific cluster.

What i think can be done is a role which limits the access of the created users to the created namespaces in the cluster?

**Comment by cwrau** [comment]
> Hm, good question.
> 
> What Roles and Rolebindings were used in the old cluster? But anyway, i think it's hard to make general assumptions about what roles are needed for a specific cluster.

This should allow for a general configuration solution not for a specific cluster nor a default set of roles. 

> What i think can be done is a role which limits the access of the created users to the created namespaces in the cluster?

What do you mean by that? 🤔

I was thinking of something like this;

```yaml
roles:
  admin:
    get:
      v1:
        - services
clusterRoles:
  admin:
    list:
      v1:
        - services
users:
  test:
    roles:
      admin:
        - kube-system
    clusterRoles:
      - admin
```

Or maybe a field `type` on the `role`? 🤔

**Comment by marvinWolff** [comment]
In the past we used a namespace "accounts" to store them. 
I think the same can be done with the `.Release.Namespace`

**Comment by cwrau** [comment]
closed by #100

**Comment by cwrau** [comment]
I be stupid
